src/app/auth/LoginContainer.ts

```diff
@@ -0,0 +1,13 @@
+import { connect } from "react-redux";
+import LoginComponent from "./LoginComponent";
+
+const mapStateToProps = state => {
+  const { loggingIn } = state.auth;
+  return {
+    loggingIn
+  };
+};
+
+const LoginContainer = connect(mapStateToProps)(LoginComponent);
+
+export default LoginContainer;
```

Comment:
And why not to export `connect` directly?

Reply:
Will change this